.eslintrc.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 13,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react",
+        "@typescript-eslint"
+    ],
+    "rules": {
```

Comment:
The idea of having these rules here is to get us to a pass right now, and then remove them and clean up the resulting errors one by one correct? If so can we just toss a comment in there explaining that?

Reply:
yes, the intention was to get the check to pass, then we can go back and remove the rule and fix as required.

Reply:
In addition to dropping a `# TODO remove rule overrides that affect code quality` perhaps we can spawn a new issue where we begin discussing which rules to turn back on? Some of these rules have greater impact than others.